apps/rewards/app/components/Styles/Header.js

```diff
@@ -0,0 +1,12 @@
+import styled from 'styled-components'
+import { Text } from '@aragon/ui'
+
+const Header = styled(Text.Block).attrs({
+  smallcaps: true,
+})`
+  color: ${({ theme }) => theme.contentSecondary};
```

Comment:
Sort of a separate discussion, but I wonder if it makes more sense to use inline `css=` props for components like this, which would let us easily stop passing in `theme` and allowing the component to define it itself:

```js
import React from 'react'
import { GU, Text, theme } from '@aragon/ui'

export default function Header({ children }) {
  const theme = useTheme()
  return (
    <Text.Block
      smallcaps
      css={`color: ${theme.contentSecondary}; margin-top: ${2 * GU}px; margin-bottom: ${GU}px`}
    >
      {children}
    </Text.Block>
  )
}
```

A trade-off is that stylelist doesn't currently inspect these inline `css=` props correctly.

What do you think?

(Also, rewriting this made me think that we should probably use GU here.)

Reply:
I like your suggestion of making this a fully-fledged component. However, I had to use `style` instead of `css`.

Reply:
I find it very strange that wrapping it in `styled()` works, but using inline `css=` doesn't. I thought they did the same thing!!

Reply:
:woman_shrugging: